pageserver/src/layered_repository/inmemory_layer.rs

```diff
@@ -369,10 +370,17 @@ impl InMemoryLayer {
 
         if old.is_some() {
             // We already had an entry for this LSN. That's odd..
-            warn!(
-                "Page version of rel {} blk {} at {} already exists",
-                self.seg.rel, blknum, lsn
-            );
+            // That is fine for checkpoint record that we update
+            // at the end of timeline WAL import.
+            match self.seg.rel {
```

Comment:
nit: can be
```rust
if !matches!(self.seg.rel, RelishTag::Checkpoint) {
 ...
}
```
maybe even add it to `if old.is_some()`

Reply:
> nit: can be
> 
> ```rust
> if !matches!(self.seg.rel, RelishTag::Checkpoint) {
>  ...
> }
> ```
> 
> maybe even add it to `if old.is_some()`

Thank you! This syntax is way better.